source/__tests__/integration/create-property.test.js

```diff
@@ -151,5 +231,9 @@ async function expect_sel_to_exist(sel) {
 }
 async function expect_value_in_sel_text(sel, value) {
```

Comment:
I'm thinking we may want to create a npm module for these type of helper functions that we use across our different projects, what do you think?

Reply:
yeah, either that or some sort of helper module class (not npm, just in the app)